docs/docs/get-started/setup-lightdash/install-lightdash.mdx

```diff
@@ -0,0 +1,175 @@
+**PGPASSWORD** | string |  | true | Postgres password
+**PGDATABASE** | string | postgres | false | Postgres database name
+**SECURE_COOKIES** | boolean | false | false | Secure cookies
+**TRUST_PROXY** | boolean | false | false | Trust the reverse proxy when setting secure cookies (via the "X-Forwarded-Proto" header)
+**LIGHTDASH_SECRET** | string | | true | This is the secret used to sign the session ID cookie and to encrypt sensitive information. **Do not share this secret!**
+**PORT** | number | 8080 | false | Port for Lightdash
+**DBT_PROJECT_DIR** | string | . | false | Path to your local dbt project. Only set this value if you want to use a dbt project from the same machine
+
+
+#### 1. Clone the Lightdash repository
+
+Clone the Lightdash code to your local machine. This will create a new directory called `./lightdash` (the Lightdash directory).
+
+```bash
+# Clone the Lightdash repo
+git clone https://github.com/lightdash/lightdash
+cd lightdash
+```
+
+#### 2. Create containers
+
+```bash
+LIGHTDASH_SECRET="not very secret" PGPASSWORD="password" docker-compose -f docker-compose.yml up --detach --remove-orphans
+```
+
+### Using docker images
+
+:::info
+
+If you're using **podman** replace `docker` with `podman` in the commands mentioned.
+
+:::
+
+#### 1. Start a Postgres container
+
+```bash
+# Example data:
+# Username: postgres
+# Password: mysecretpassword
+# Database name: postgres
+# Port: 5432
+
+docker run --name lightdash-db -p "5432:5432" \
+    -e POSTGRES_PASSWORD=mysecretpassword -d postgres
+```
+
+#### 2. Start Lightdash container
+
+Lightdash environment variables:
+
+Name | Type | Default | Required | Description
+---|---|---|---|---
+**PGHOST** | string |  | true | Postgres hostname
+**PGPORT** | number |  | true | Postgres port
+**PGUSER** | string |  | true | Postgres user
+**PGPASSWORD** | string |  | true | Postgres password
+**PGDATABASE** | string |  | true | Postgres database name
+**LIGHTDASH_SECRET** | string | | true | This is the secret used to sign the session ID cookie and to encrypt sensitive information. **Do not share this secret!**
+**SECURE_COOKIES** | boolean | false | false | Secure cookies
+**TRUST_PROXY** | boolean | false | false | Trust the reverse proxy when setting secure cookies (via the "X-Forwarded-Proto" header)
+:::info
+
+If you want to use a dbt project from the same machine, you will need to set a volume to **/usr/app/dbt**. Example:
+`-v ~/my/dbt/path:/usr/app/dbt`
+
+:::
+
+```bash
+# Example data:
+# DB Host: 192.168.0.1
+# DB Port: 5432
+# DB Username: postgres
+# DB Password: mysecretpassword
+# DB name: postgres
+# Lightdash secret: not very secret
+# Lightdash port: 8080
+
+docker run --name lightdash-app \
+    -e PGHOST=192.168.0.1 -e PGPORT=5432 \
+    -e PGUSER=postgres -e PGPASSWORD=mysecretpassword \
+    -e PGDATABASE=postgres -e LIGHTDASH_SECRET="not very secret" \
+    -p 8080:8080 \
+    -d lightdash/lightdash:latest
+```
```

Comment:
```suggestion
```

----

## Getting set up on Lightdash Cloud

Want to have a production-ready Lightdash instance, but not comfortable with self-hosting? You can [sign up to our waitlist for Lightdash Cloud.](https://hamzah905194.typeform.com/to/HFlicx4i)

With Lightdash Cloud, you get a Lightdash instance that's production-ready (so you can share all your insights with your team) but you don't have to worry about any of the setup for hosting the instance yourself. So, if you're not too sure about all of this container, Heroku and cloud-platforms business, or you're just not interested in the overhead of hosting your BI yourself, then [Lightdash Cloud](https://hamzah905194.typeform.com/to/HFlicx4i) might be a good option for you :)
```